kick/http.py

```diff
@@ -236,22 +237,21 @@ async def request(self, route: Route, **kwargs) -> Any:
                 f"Making request to {route.method} {url}. headers: {headers}, params: {kwargs.get('params', None)}, json: {kwargs.get('json', None)}"
             )
             try:
-                res = await self.__session.request(
+                res = await requests_ORI.AsyncSession().request(
                     route.method,
-                    url
-                    if self.whitelisted is True
```

Comment:
If you're going to remove generating the url like this, you should also remove `HTTPClient.whitelisted`, and the others. `kick.Client`'s docstring would also need to change